src/lib/viewers/image/ImageBaseViewer.js

```diff
@@ -137,6 +138,19 @@ class ImageBaseViewer extends BaseViewer {
         this.emit('panstart');
     }
 
+    /**
+     * Determines whether discoverability is enabled
+     *
+     * @private
+     * @return {boolean} value of whether discoverability is enabled for given type
+     */
+    isDiscoverabilityEnabled() {
+        const { experiences = {} } = this.options;
+        const canShow = Object.values(experiences).some(experience => experience.canShow);
```

Comment:
iirc, annotations is the only experience we're passing through right now, but should this be so generic? What happens if there is a new preview thumbnails siderail experience (which doesn't apply to images) would that prevent image discoverability?

Reply:
@ConradJChan - That is a good point. I think it's most likely if any experience can show we would want to disable discoverability. I also think it would be more likely to forget to add than remember to add it, if we created some sort of eligible experiences list. I think it would make sense to address this, when the time comes, but open to make the suggested change. Let me know what you think!

Reply:
I'll go ahead and merge this PR and we can make updates in a separate PR if needed!